modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/client/EventedExtensionsClientDecorator.java

```diff
@@ -58,4 +61,11 @@ public <T, A> CompletableFuture<Result<T, A>> execute(String extension,
       }
     });
   }
+
+  @Override
+  public <T, A> SourceHandler createSource(String extension, String sourceName,
+                                           Consumer<SourceResultCallback<T, A>> callback,
+                                           Consumer<SourceParameterizer> parameters) {
+    return extensionsClient.createSource(extension, sourceName, callback, parameters);
```

Comment:
The context of the event will not be used to create the source.

What about logging a Warning or a throwing a not supported exception here?

Reply:
that's an internal implementation detail about a class that's not even API and only used for operations. What value would such log add?